src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java

```diff
@@ -34,12 +35,19 @@ private void validateSavedDataOld() {
         testResultsSteps.validateData((TestResultsGet) vehicleCancelledDataOld.build());
     }
 
+    private String test_results_cancelled_json;
```

Comment:
line 20 unused import

Also looks like validateSavedDataOld is no longer being used, is it still required?